package.json

```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "antd": "^3.15.1",
     "axios": "^0.18.0",
     "node-sass": "^4.11.0",
     "react": "^16.8.1",
     "react-dom": "^16.8.1",
+    "react-infinite-scroller": "^1.2.4",
```

Comment:
Love the resourcefulness using libraries, though I'd have preferred seeing your own logic for the infinite scroller :P